OperatingSystem-Services/Platform-Linux/OpenVAS/README.md

```diff
@@ -0,0 +1,419 @@
+                image: registry.community.greenbone.net/community/ospd-openvas:latest
+                restart: on-failure
+                hostname: ospd-openvas.local
+                cap_add:
+                    - NET_ADMIN # for capturing packages in promiscuous mode
+                    - NET_RAW # for raw sockets e.g. used for the boreas alive detection
+                security_opt:
+                    - seccomp=unconfined
+                    - apparmor=unconfined
+                command:
+                    [
+                    "ospd-openvas",
+                    "-f",
+                    "--config",
+                    "/etc/gvm/ospd-openvas.conf",
+                    "--notus-feed-dir",
+                    "/var/lib/notus/advisories",
+                    "-m",
+                    "666"
+                    ]
+                volumes:
+                    - gpg_data_vol:/etc/openvas/gnupg
+                    - vt_data_vol:/var/lib/openvas/plugins
+                    - notus_data_vol:/var/lib/notus
+                    - ospd_openvas_socket_vol:/run/ospd
+                    - redis_socket_vol:/run/redis/
+                    - openvas_data_vol:/etc/openvas/
+                    - openvas_log_data_vol:/var/log/openvas
+                depends_on:
+                    redis-server:
+                    condition: service_started
+                    gpg-data:
+                    condition: service_completed_successfully
+                    vulnerability-tests:
+                    condition: service_completed_successfully
+                    configure-openvas:
+                    condition: service_completed_successfully
+
+                gvm-tools:
+                image: registry.community.greenbone.net/community/gvm-tools
+                volumes:
+                    - gvmd_socket_vol:/run/gvmd
+                    - ospd_openvas_socket_vol:/run/ospd
+                depends_on:
+                    - gvmd
+                    - ospd-openvas
+
+            volumes:
+                gpg_data_vol:
+                scap_data_vol:
+                cert_data_vol:
+                data_objects_vol:
+                gvmd_data_vol:
+                psql_data_vol:
+                vt_data_vol:
+                notus_data_vol:
+                psql_socket_vol:
+                gvmd_socket_vol:
+                ospd_openvas_socket_vol:
+                redis_socket_vol:
+                openvas_data_vol:
+                openvas_log_data_vol:
+
+            secrets:
+                server-certificate:
+                    file: /home/blueteam/docker_keys/servercert.pem
+                private-key:
+                    file: /home/blueteam/docker_keys/serverkey.pem
+        </details>
+
+5. **Starting The Docker Containers**
+    
+    > [!NOTE] This process could take a long time (Possibly several hours) depending on your machine's resources. Consider using tmux to run commands in order to save and come back.
```

Comment:
Probably modify this to IMPORTANT since it is a good note, but we want them to stand out more and it is quite important they know this will take a long time.
```suggestion
    > [!IMPORTANT] This process could take a long time (Possibly several hours) depending on your machine's resources. Consider using tmux to run commands in order to save and come back.
```